.circleci/config.yml

```diff
@@ -53,12 +53,14 @@ jobs:
       - restore_cache:
           key: *cache_key
 
+      - run: sudo cp .circleci/bazel.rc /etc/bazel.bazelrc
+
       # Helpful for debugging, so you can check you have the same bazel version when you need
       # to reproduce a failure.
       - run: bazel info release
 
       # Install the dependencies from NPM
-      - run: bazel run --config=ci @yarn//:yarn install
+      - run: bazel run --config=ci @nodejs//:yarn install
```

Comment:
like the rules_typescript PR - remove `--config=ci` everywhere and remove the `build:ci` lines from `tools/bazel.rc`

Reply:
👍